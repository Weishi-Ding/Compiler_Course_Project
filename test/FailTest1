int main () {
    /*semantic negative case 1: binop of different types*/
    int a = 2;
    double b = 2.2;
    print(a * b);
}