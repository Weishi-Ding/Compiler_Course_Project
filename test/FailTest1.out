Fatal error: exception Failure("illegal binary operator int * double in a * b")